### ISSUE TITLE
GameGuru MAX - Request - Add Weapon Import to Importer

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#5090]
Currently, when attempting to use a custom weapon, we are required to copy a weapon folder that's of a similar weapon type (hammer, saw, colt pistol, etc.), rename it, remove the old DBO and associated texture files, import our weapon into MAX to generate a DBO file, copy in our newly created DBO file over and and make sure it's named properly for both the weapon we want (crowbar, etc.) and also make a copy named weapon.dbo, etc. Once again, we are 'forced' to deal with MAX's folder structure, edit text files, and do all this stuff manually outside of the MAX editor. 

Instead, I'd like to see a weapon importer that automates most (if not all) of this process. When you go to import, you would tick or select that the imported model is a weapon. After that, you would tick or select the kind of weapon (like a hammer, like a shovel, like a pistol, etc.), and you would name it as you would any other model you import into MAX. Once you click 'Import', MAX would automatically create a folder in the correct place, naming it after your weapon for you; create both the weapon.dbo with textures named correctly and the DBO for your weapon (such as crowbar.dbo) with it's associated textures, update the HUD.dbo file (or whatever needs to happen for everything to work correctly in MAX). Etc.

Basically, let's let MAX create all this stuff for us, automating this, so it's really not any more difficult for the end-user to import a weapon in MAX than it is to import any other model into MAX.

The gunspect file should be accessible somehow from MAX, too, so that the end-user can access it without having to dig through folders to find it. This could be as simple as selecting the weapon in the MAX editor (or if equipped by the player) and getting a button on the right-side properties panel that can be clicked to open the gunspect text file ... or maybe the pertinent  stuff that can be changed could simply shows up there, like things do when selecting a behavior? Point being, make it easy for the end-user to access and make changes to things like weapons without having to leave MAX and find the correct folder and file.

The weapon import could also have a tick box for 'use new arms' or whatever so that this is done automatically as well.

**Comment by mav3r1ck198-1** [comment]
A freaking THOUSAND Times yes.

**Comment by LeeBamberTGC** [comment]
@MonkeyFrogStudio Yes its a longer-winded process than having a 'weapon builder' feature.  You do not need to leave MAX, just reload the level as that will trigger new weapons to be reloaded when running the test level.  I usually move the start marker, click BACK, select NO, then when in storyboard, click the level again and when I return, I can click test level and the new weapon changes show up.  Once bugs are out of the way, my idea to move this whole weapon making process into the model importer, or add gunspec to the hotswap system so 'any' change triggers weapon reloads.

**Comment by MonkeyFrogStudio** [comment]
Thanks, @LeeBamberTGC . I will give that a try to test gunspec changes. I appreciate it.

**Comment by MonkeyFrogStudio** [comment]
Had requested something similar, though not as detailed here:

https://github.com/TheGameCreators/GameGuruRepo/issues/4855

I closed that one in favor of this one.
